src/history/git.js

```diff
@@ -6,7 +6,7 @@ import simpleGit from 'simple-git';
 
 const __dirname = path.dirname(new URL(import.meta.url).pathname);
 
-const DATA_PATH = '../../data';
+const DATA_PATH = process.env.CI ? '../../CGUs-data': '../../../CGUs-data';
```

Comment:
> I don't really understand why we need this, and I'm really not a fan of having production code that does disjunctions based on CI / tests environments 😕

Reply:
Yep, I undestand. I'm already fixing this in another branch with the config module